src/js/components/Router.jsx

```diff
@@ -224,6 +229,7 @@ const Router = (props) => {
             <MainLayoutRoute path="**/stockMovement/createOutbound/:stockMovementId?" component={AsyncStockMovement} />
             <MainLayoutRoute path="**/stockMovement/importOutboundStockMovement" component={AsyncOutboundImport} />
             <MainLayoutRoute path="**/inventory/cycleCount/count" component={AsyncCycleCountCountStep} />
+            <MainLayoutRoute path="**/inventory/cycleCount/resolve" component={AsyncCycleCountResolveStep} />
```

Comment:
[Discussion] - If I understand this correctly there are no steps involved there, perhaps it would be better to rename "steps" into "forms"?

Reply:
I treated it as the next step of the selection (after to count tab)